lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -396,6 +424,72 @@ def test_rescoring_randomized_problem(self):
         for user in self.users:
             self.check_state(user, descriptor, 0, 1, expected_attempts=2)
 
+    @patch('lms.djangoapps.instructor_task.tasks_helper.tracker')
+    @patch('lms.djangoapps.grades.signals.handlers.tracker')
+    @patch('lms.djangoapps.grades.models.tracker')
+    def test_rescoring_events(self, grades_tracker, handlers_tracker, instructor_task_tracker):
```

Comment:
I suggest moving this test to the grade's test_integration.py file and also verifying that the events for the original submission of the problem is as expected in addition to the events after restoring the problem.  What do you think?